index.html

```diff
@@ -9,11 +9,66 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+<body>
+    <header>
+       
+    <!--Navigation--> 
+    <nav>    
+        <a href="#" class="logo"> 
+           <img src="img/karma-logo.svg" alt="logo">
+        </a>     
+        <a href="">Meet Karma</a>
+        <a href="">How it works</a>
+        <a href="">Store</a>
+        <a href="">Blog</a>
+        <a href="">Help</a>
+        <a href="">Log in</a>
+    </nav>
+
+    <!--Hero-->
+    <section class="hero-image"></section>
+        <div class="hero-text">
+            <H1>Introducing<br> Carma</H1>
+            <p>Bring wifi with you,<br>everywhere you go.</p>
+            <button>Learn More</button>
+        </div>
+    </section>
+</header>
+
+    <!--Main-->
+<main>
+    <h2>Everyone needs a little Karma.</h2>
+    <article>
+     <div>
+        <img src="img/icon-devices.svg">
```

Comment:
```suggestion
        <img src="img/icon-devices.svg" alt="Various computer devices">
```
`img` elements should have an `alt` description for screen reader users. Same applies to the rest of the images